### ISSUE TITLE
Rename action modifies comment and string words equal to Symbol text

### ISSUE THREAD

**Issue by Myrslaver** [razzmatazz/csharp-language-server#157]
Hello,
The commit 65cd66f changed the SymbolRenameOptions passed to the RenameSymbolAsync function from this

```f#
SymbolRenameOptions(RenameOverloads=true, RenameFile=true)
```

to this

```f#
SymbolRenameOptions(RenameOverloads = true, RenameInStrings = true, RenameInComments = true)
```

Which it means that every string and comment inside the solution/projects, that contains a word equal to the Symbol text, is renamed to the text of the action input.
The file which contains this modification is this one: [Rename.fs](https://github.com/razzmatazz/csharp-language-server/blob/master/src/CSharpLanguageServer/Handlers/Rename.fs), line 170.
I think this is a bug, and it should be reverted to the original.
This project is so cool btw, thank you for your work!

**Comment by razzmatazz** [comment]
I concur, old behaviour restored in https://github.com/razzmatazz/csharp-language-server/commit/ccee0efa44c5166e644265653b543a5a2e0fcbf0

**Comment by razzmatazz** [comment]
I believe this has been fixed in 0.14.0, please reopen if not so.
